packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md

```diff
@@ -95,16 +95,105 @@ See [Customization examples](#customization-examples) for snippets that you can
+       const submit = formBag.uischema.elements[submitIndex];
+       submit.label = 'Register';
+   }
+});
+```
+
+#### Remove an unused link example
+
+The following example shows how to remove the **Help**, **Unlock account?**, and **Forgot password?** links from the **Identify** page:
+
+```javascript
+oktaSignIn.afterTransform('identify', ({ formBag }) => {
+ const help = formBag.uischema.elements.find(ele => ele.type === 'Link' && ele.options.dataSe === 'help');
+ const unlock = formBag.uischema.elements.find(ele => ele.type === 'Link' && ele.options.dataSe === 'unlock');
+ const forgot = formBag.uischema.elements.find(ele => ele.type === 'Link' && ele.options.dataSe === 'forgot-password');
+ formBag.uischema.elements = formBag.uischema.elements.filter(ele => ![help, unlock, forgot].includes(ele));
+});
+```
+
+#### Add an instructional paragraph example
+
+The following example shows how to add a custom description to the **Password recovery** page:
+
+```javascript
+oktaSignIn.afterTransform?.('identify-recovery', ({ formBag }) => {
+   const titleIndex = formBag.uischema.elements.findIndex(ele => ele.type === 'Title');
+   // Add custom description after title
+   const descr = {
+       type: 'Description',
+       contentType: 'subtitle',
+       options: {
+           variant: 'body1',
+           content: '<div class=\'my-reset-description\'>Description<br />about<br />recovery</div>'
+       },
+   };
+   if (titleIndex != -1) {
+       formBag.uischema.elements.splice(titleIndex + 1, 0, descr);
+   }
+});
+```
+
+### Use the afterRender function
+
+For the third generation Sign-In Widget, the [`afterRender`](https://github.com/okta/okta-signin-widget?tab=readme-ov-file#afterrender) function doesn't work when used to make DOM manipulations and other render-related side effects.
+
+If you use `afterRender` for DOM manipulations, the Okta Sign-In Widget reverts any customizations to default settings. See [Components and Hooks must be pure](https://react.dev/reference/rules/components-and-hooks-must-be-pure).
+
+* To use `afterRender` for DOM manipulations, consider using the `MutationObserver` function. See [Use MutationObserver for Dom manipulations](#use-mutationobserver-for-dom-manipulations).
+* To use `afterRender` for non-DOM manipulations, you don't need the `MutationObserver` function. See [Use afterRender for non-DOM manipulations](#user-afterrender-for-non-dom-manipulations).
 
 #### Use MutationObserver for DOM manipulations
 
-To prevent the third generation of the Sign-In Widget from reverting your `afterRender` customizations, use the DOM `MutationObserver` function. See [MutationObserver](https://dom.spec.whatwg.org/#ref-for-dom-mutationobserver-mutationobserver).
+To prevent the third generation Sign-In Widget from reverting your `afterRender` customizations, use the DOM `MutationObserver` function. See [MutationObserver](https://dom.spec.whatwg.org/#ref-for-dom-mutationobserver-mutationobserver).
```

Comment:
```suggestion
To prevent the third-generation Sign-In Widget from reverting your `afterRender` customizations, use the DOM `MutationObserver` function. See [MutationObserver](https://dom.spec.whatwg.org/#ref-for-dom-mutationobserver-mutationobserver).
```